tests/test_functions.cpp

```diff
@@ -80,6 +80,9 @@ NB_MODULE(test_functions_ext, m) {
     });
 
     /// Test call_guard feature
+    m.def("test_call_guard_wrapper_rvalue_ref", [](int&& i) { return i; },
```

Comment:
Does that test really fail if the `forward` is missing? It does not seem to me that way.

Reply:
I wrote the test first and checked that it failed, just to make sure the problem is not on my end. It only fails when call_guard is present, though.